platform/docs/docs/platform/services/customization-service/customizationService.md

```diff
@@ -0,0 +1,467 @@
+        ```
+
+
+### 2. **Mode Scope**
+   - **Purpose**: Apply customizations specific to a particular mode.
+   - **Lifecycle**: These customizations are cleared or reset when switching between modes.
+   - **Example**: Use the `setCustomizations` method to define mode-specific behavior.
+     ```tsx
+     customizationService.setCustomizations({
+       'studyBrowser.sortFunctions': {
+         $set: [
+           {
+             label: 'Mode-Specific Sort Function',
+             sortFunction: (a, b) => b.SeriesDate - a.SeriesDate,
+           },
+         ],
+       },
+     });
+     ```
+
+
+
+### 3. **Global Scope**
+   - **Purpose**: Apply system-wide customizations that override both default and mode-scoped values.
+   - **How to Configure**:
+     1. Add global customizations directly to the application's configuration file:
+        ```jsx
+        window.config = {
+          name: 'config/default.js',
+          routerBasename: '/',
+          customizationService: [
+            {
+              'studyBrowser.sortFunctions': {
+                $push: [
+                  {
+                    label: 'Global Sort Function',
+                    sortFunction: (a, b) => b.SeriesDate - a.SeriesDate,
+                  },
+                ],
+              },
+            },
+          ],
+        };
+        ```
+
+     2. Use Namespaced Extensions:
+        - Instead of directly specifying customizations in the configuration, you can refer to a predefined customization module within an extension:
+
+        ```jsx
+        window.config = {
+          name: 'config/default.js',
+          routerBasename: '/',
+          customizationService: [
+            '@ohif/extension-cornerstone.customizationModule.newCustomization',
+          ],
+        };
+        ```
+
+        - In this example, the `newCustomization` module within the `@ohif/extension-cornerstone` extension contains the global customizations. The application will load and apply these settings globally.
+
+          ```tsx
+          function getCustomizationModule() {
+            return [
+              {
+                name: 'newCustomization',
+                value: {
+                  'studyBrowser.sortFunctions': {
+                    $push: [
+                      {
+                        label: 'Global Namespace Sort Function',
+                        sortFunction: (a, b) => b.SeriesDate - a.SeriesDate,
+                      },
+                    ],
+                  },
+                },
+              },
+            ];
+          }
+          ```
+
+
+### Priority of Scopes
+When a customization is retrieved:
+1. **Global Scope**: Takes precedence if defined.
+2. **Mode Scope**: Used if no global customization is defined.
+3. **Default Scope**: Fallback when neither global nor mode-specific values are available.
+
+
+As you have guessed the `.setCustomizations` accept a second argument which is the scope. By default it is set to `mode`.
+
```

Comment:
Can you add a section in here explaining how to build up customization modules across multiple extensions?  For example, to add sop class handlers to a set of sop class handlers, or to add buttons to a list of buttons.
Then, show how the mode can "register" those during the mode startup, or use the compound registrations to do things.

Reply:
I thought i did, but i will expand on it